src/main/java/org/folio/linked/data/integration/kafka/sender/inventory/InstanceReplaceMessageSender.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.linked.data.integration.kafka.sender.inventory;
+
+import static org.folio.linked.data.util.Constants.FOLIO_PROFILE;
+
+import java.util.Collection;
+import lombok.RequiredArgsConstructor;
+import org.folio.linked.data.integration.kafka.sender.ReplaceMessageSender;
+import org.folio.linked.data.model.entity.Resource;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.util.Pair;
+import org.springframework.stereotype.Service;
+
+@Service
+@Profile(FOLIO_PROFILE)
+@RequiredArgsConstructor
+public class InstanceReplaceMessageSender implements ReplaceMessageSender {
```

Comment:
Do you suggest not to implement 'apply' and 'accept' methods of Consumer/BiFunction interfaces?
Do you mean to get rid of them?
Do you mean to get rid of 'produce' method in ReplaceMessageSender interface?

So that's not clear at all. Could you please write self-explanatory comment?

Reply:
discussed, fixed